Presentation.Web/app/services/gridStateService.ts

```diff
@@ -230,9 +172,29 @@
                     // note the order the options are merged in (below) is important!
                     options = <IGridSavedState> _.merge({}, localOptions, profileOptions);
                 }
-                else if (orgOptions) {
-                    options = <IGridSavedState>_.merge(options, localOptions, orgOptions);
+
+                if ($window.sessionStorage.getItem(organizationalConfigurationVersionKey) !== null) {
+                    // Session updates has not changed the grid as updates to the grid which changes the columns causes the version to be deleted
+                    // So we use the local organization configuration if it exists
+                    if (orgStorageColumns) {
+                        var columns: { [persistId: string]: { index: number; width: number, hidden?: boolean } } = {};
+
+                        var gridColumnWidths: { [persistId: string]: {width: number } } = {};
+
+                        // We need to iterate over all columns to hide them. During the iteration we also store the column widths in a map
+                        grid.columns.forEach(x => {
+                            x.hidden = true;
+                            gridColumnWidths[x.persistId] = { width: x.width as number };
+                        });
```

Comment:
Laver mappet på denne måde da jeg er nød til at skulle iterere over emnerne for at skjule dem